### DISCUSSION TITLE
image(tw, height="chi", leadingspecies=TRUE, reorder=TRUE)，How to add legends to dendrograms and heatmaps of plots and species clusters？

### DISCUSSION THREAD

**Discussion by chaijian2019** [jarioksa/twinspan#5] [Q&A]
How to add legends to dendrograms and heatmaps of plots and species clusters？what is the yellow and red? image(tw, height="chi", leadingspecies=TRUE, reorder=TRUE)
![image](https://github.com/user-attachments/assets/17bcce6b-6acc-44e4-83c0-c2a252df5ec3)

**Comment by jarioksa** [comment]
I assume you ask specifically about `image` function in `twinspan`?

The function is a case of standard **R** function `graphics::image`, and accepts the arguments of that function. Use `help(image, graphics)` to see how to modify the function. As to axis labels, that help describes arguments `xlab` and `ylab` which will add labels to axes.

The colours are related to the averages of within species &times; quadrat groups or species group &times; quadrat group cells. With this colour palette, the redder the higher mean. Again, consult `help(image, graphics)` for modifying or setting the colour palettes.

**Reply by chaijian2019** [reply]
Thank you for sharing and your reply. Your reply has solved many of my problems.
